collection-scripts/gather_serving

```diff
@@ -2,16 +2,12 @@
 # shellcheck disable=SC1091
 source common.sh
 resources=("inferenceservices" "inferencegraphs" "trainedmodels" "servingruntimes" "clusterstoragecontainers" "predictors" "localmodelnodegroups")
-# Dependent resources ossm
-resources+=("smcp" "smm" "smmr" "gateways.networking.istio.io" "virtualservices" "authorizationpolicies" "envoyfilters" "destinationrules")
-# Dependent resources knative
-resources+=("knativeservings" "configurations" "routes.serving.knative.dev" "services.serving.knative.dev" "revisions")
 # Dependent resources authorino
 resources+=("authconfigs" "authorinos" "authpolicies.kuadrant.io")
 # Dependent resources NIM
 resources+=("accounts.nim.opendatahub.io")
 # Dependent resources llm-d
-resources+=("llminferenceserviceconfigs" "llminferenceservices" "leaderworkersets" "gatewayclasses" "gateways.gateway.networking.k8s.io" "httproutes" "inferencepools")
```

Comment:
can we also capture `leaderworkersets` ?

Reply:
sure, if it is needed i can put it back

Reply:
Ah I see they are part of the other script

Reply:
however, in this way, they cannot capture only the serving parts and get the full picture